liota/dccs/iotcc.py

```diff
@@ -83,15 +83,16 @@ def on_response(msg):
                     return True
                 else:
                     log.debug("Processed msg: {0}".format(json_msg["type"]))
-                    on_response(self.recv_msg_queue.get())
+                    on_response(self.recv_msg_queue.get(True,10))
```

Comment:
It is a general logic in the lower usecase you`ll need retry for create_relationship over here to receive the other messages for HelixProtocol.